src/Microsoft.Framework.Runtime/Servicing/Breadcrumbs.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.Framework.Runtime.Servicing
     public class Breadcrumbs
     {
         // It is recommended that you disable breadcrumbs only for perf runs
-        private const string NoBreadcrumbsEnvironmentVariableName = "DNX_NO_BREADCRUMBS";
+        private const string NoBreadcrumbsEnvironmentVariableName = Constants.RuntimeShortName + "_NO_BREADCRUMBS";
```

Comment:
Nit: this is still `dnx_NO_BREACRUMBS` and not `DNX_NO_BREADCRUMBS`. Consider `Constants.RuntimeShortName.ToUpper() + "_NO_BREADCRUMBS"`. May require using `static readonly` instead of `const`